huaweicloud/resource_huaweicloud_vpc_eip.go

```diff
@@ -307,6 +307,8 @@ func resourceVpcEIPV1Read(d *schema.ResourceData, meta interface{}) error {
 	if vpcV2Client, err := config.NetworkingV2Client(GetRegion(d, config)); err == nil {
 		if resourceTags, err := tags.Get(vpcV2Client, "publicips", d.Id()).Extract(); err == nil {
 			tagmap := utils.TagsToMap(resourceTags.Tags)
+			// ignore "CCE-Dynamic-Provisioning-Node"
```

Comment:
Comments should express ’Why‘ rather than ‘What’.

Reply:
fixed